### PR TITLE
HARVESTER: fix vm unit display error

### PR DESCRIPTION
**Pull request by WuJun2016**
# links
- https://github.com/harvester/harvester/issues/1534
- https://github.com/harvester/harvester/issues/1429
- https://github.com/harvester/harvester/issues/1550

Note: `memory.guest` is calculated by the backend

### CODE REVIEW

**Comment by weihanglo** [comment]
It would be better if vm template version gets the same behaviour!
 https://github.com/harvester/harvester/pull/1554#issuecomment-971615050

**Comment by WuJun2016** [comment]
Thanks, the template has been synchronized and updated

File: detail/kubevirt.io.virtualmachine/VirtualMachineTabs/VirtualMachineBasics.vue [hunk 1]
```diff
@@ -92,7 +92,7 @@ export default {
     flavor() {
       const domain = this.value?.spec?.template?.spec?.domain;
 
-      return `${ domain.cpu.cores } vCPU , ${ domain.resources.requests.memory } Memory`;
+      return `${ domain.cpu.cores } vCPU , ${ domain.resources.limits.memory } Memory`;
```
Comment (n313893254) [inline detail/kubevirt.io.virtualmachine/VirtualMachineTabs/VirtualMachineBasics.vue:95]: Is the "Memory" should be i10n?

**Review by n313893254 — CHANGES_REQUESTED** [review]

**Comment by WuJun2016** [comment]
Done

**Review by n313893254 — APPROVED** [review]

**Pull request merged**